src/pip/_internal/pep425tags.py

```diff
@@ -252,10 +252,10 @@ def get_supported(versions=None, noarch=False, platform=None,
         abis[0:0] = [abi]
 
     abi3s = set()
-    import imp
-    for suffix in imp.get_suffixes():
-        if suffix[0].startswith('.abi'):
-            abi3s.add(suffix[0].split('.', 2)[1])
+    from pip._internal.utils.compat import get_extension_suffixes
```

Comment:
This import can happen at the top of the file.

Reply:
Done.